src/OrchardCore.Modules/OrchardCore.Flows/Views/BagPart.Edit.cshtml

```diff
@@ -57,12 +57,12 @@
         </div>
     </div>
 
-
+    
```

Comment:
Remove extra whitespace

Reply:
Seems like I should stop using VS2019, and just use VS Code